packages/react-native-lockdown/src/index.js

```diff
@@ -0,0 +1,200 @@
+'use strict'
+
+/**
+ * React Native lockdown entry point.
+ *
+ * @packageDocumentation
+ */
+
+/**
+ * @typedef {import('@react-native/metro-config').SerializerConfig} SerializerConfig
+ */
+
+/**
+ * @typedef {(moduleId: number | string) => string} GetRunModuleStatement
+ * @see {@link https://metrobundler.dev/docs/configuration/#getrunmodulestatement}
+ */
+
+/**
+ * @typedef {(options: { platform?: string }) => ReadonlyArray<string>} GetPolyfills
+ * @see {@link https://metrobundler.dev/docs/configuration/#getpolyfills}
+ */
```

Comment:
Were you planning on adding a `tsconfig.json` so that we can ship type declarations?  If not, I can send a PR to do this after.  If so, please lmk if you run into trouble (since our setup is very "composite project"-heavy)

Reply:
yes after merging 🙏 for consistent TS setup/shipping in all our packages, will do